viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -233,6 +235,20 @@ Ext.define("viewer.components.Edit", {
                 xtype: "container",
                 items: this.createEditButtons()
             },
+            {
+                xtype: 'sliderfield',
+                itemId: 'rotateSlider',
+                label: 'rotatie',
+                value: 0,
+                minValue: 0,
+                maxValue: 360,
```

Comment:
is het een idee om minValue -259, value 0 en maxValue 259 te maken? dan kun je links en rechtsom draaien

Reply:
Kan... in principe kan je nu ook een volledig rondje draaien.
Maakt mij niet veel uit

Reply:
Wat vinden @mtoonen @KJLammers ?

Reply:
Lijkt me wel netjes om zowel links- als rechtsom te kunnen draaien, ookal kan je formeel hetzelfde